src/apps/settings/src/app/views/Instance/Instance.js

```diff
@@ -195,31 +205,61 @@ export default connect((state) => {
               } else if (field.key === "preferred_domain_prefix") {
                 return (
                   <div key={field.ZUID} className={styles.column}>
-                    <FieldTypeBinary
-                      key={field.ZUID}
-                      name={field.key}
+                    <FormLabel sx={{ color: "primary.dark" }}>
+                      <Stack
+                        direction="row"
+                        alignItems="center"
+                        sx={{
+                          my: 1,
+                        }}
+                      >
+                        <Tooltip
+                          title={`Activating the WWW setting requires DNS setup of both the apex domain and www sub-domain.`}
+                        >
+                          <InfoIcon fontSize="small" sx={{ mr: 1 }} />
```

Comment:
If you are using Stack might as well use the spacing attribute as opposed to margins